user/utils.py

```diff
@@ -0,0 +1,14 @@
+import datetime
+
+import jwt
+from condominium_manager_server.settings import SECRET_JWT
+
+
+def generate_access_token(user):
+    access_token_payload = {
+        'user_id': user.id,
+        'exp': datetime.datetime.utcnow() + datetime.timedelta(days=0, minutes=5)
+    }
```

Comment:
**Suggestion:** The 'generate_access_token' function could be improved by handling the case where the 'user' argument is None. Currently, if 'user' is None, the function will raise an AttributeError when trying to access 'user.id'.
```suggestion
def generate_access_token(user):
    if user is None:
        raise ValueError("User is None")
    access_token_payload = {
        'user_id': user.id,
        'exp': datetime.datetime.utcnow() + datetime.timedelta(days=0, minutes=5)
    }
```